Crashlytics/Crashlytics/Models/Record/FIRCLSReportAdapter.m

```diff
@@ -230,8 +232,10 @@ - (google_crashlytics_Platforms)protoPlatformFromString:(NSString *)str {
  */
 pb_bytes_array_t *FIRCLSEncodeData(NSData *data) {
   pb_bytes_array_t *pbBytes = malloc(PB_BYTES_ARRAY_T_ALLOCSIZE(data.length));
-  memcpy(pbBytes->bytes, [data bytes], data.length);
-  pbBytes->size = (pb_size_t)data.length;
+  if (pbBytes != NULL) {
+    memcpy(pbBytes->bytes, [data bytes], data.length);
+    pbBytes->size = (pb_size_t)data.length;
+  }
```

Comment:
Same here - can we just return NULL when pbBytes == null?

Reply:
Done.

Reply:
Done.